config/other.go

```diff
@@ -220,3 +219,18 @@ func setDefaults(conf *Config, bus awlevent.Bus) {
 		conf.Update.TrayAutoCheckInterval = "24h"
 	}
 }
+
+func stringSlicesEqual(s1, s2 []string) bool {
+	if len(s1) != len(s2) {
+		return false
+	}
+	sort.Strings(s1)
+	sort.Strings(s2)
+	for i := range s1 {
+		if s1[i] != s2[i] {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
Hm, since it is used only here, I guess, it is ok. But I want to share with you my (im)HO. So, as I think, function name should make it clear, that slices will be sorted, and that we talk about content equality and ignore oder

Reply:
You are right, it's a bit strange that function sorts slices in-place and therefore changes config.
Added comments to function and now it sorts copy of slices. Not the most efficient solution, but should be OK for this case